recipes/ncbi-vdb/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "2.9.0" %}
+{% set sha256 = "b4099e2fc3349eaf487219fbe798b22124949c89ffa1e7e6fbaa73a5178c8aff" %}
+
+package:
+  name: ncbi-vdb
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  fn: ncbi-vdb-{{ version }}.tar.gz
+  url: https://github.com/ncbi/ncbi-vdb/archive/{{ version }}-1.tar.gz
+  sha256: {{ sha256 }}
+
+
+requirements:
+  build:
+    - gcc # [not osx]
+    - llvm # [osx]
+    - perl
+    - libiconv
+    - libxml2
+    - ncbi-ngs-sdk
+    - bison
+
+  run:
+    #
```

Comment:
No runtime dependencies? Are these now filled automatically? I somehow doubt the library will work without at least `libxml2`, `libiconv` and `libgcc`.

Reply:
We link against `libxml2` when found, but it is not required. In fact, its only use is as a relic for some very old software that is no longer supported. We also detect `libhdf5` and include some additions for that if found, in order to support a loader that is only run within NCBI. We link against `-ldl`, `-lm` and `-lpthreads`.